Quality_Assurance.md

```diff
@@ -0,0 +1,55 @@
+# Quality Assurance
+
+DataJoint and DataJoint Elements serve as a framework and a starting point for numerous new projects; setting the standard of quality for data architecture and software design. To ensure higher quality, the following policies have been adopted into the software development lifecycle (SDLC).
+
+## Coding Standards
+
+When writting code, the following principles should be observed.
+
+- **Style**: Code shall be written for clear readability. Uniform and clear naming conventions, module structure, and formatting requirements shall be established across all components of the project. Python's [PEP8](https://www.python.org/dev/peps/pep-0008/#naming-conventions) standard offers much guidance to this regard which can similarly be applied to all languages.
```

Comment:
good start, but would be good to fill in some DataJoint specific standards:

- attribute names being unique and having classname prefix (e.g. foo_time foo_id vs time / id)
- module-per-schema
- classes in the singular (e.g. Subject vs Subjects)
- master-part vs top-level fkeys guidelines
- decimals vs floats
- descriptive pkeys vs synthetic pkeys

Also, pep8 has 79 column clean requirement. Can almost guarantee we are not following this.

perhaps should be refiled as followups, or can PR against this PR if / when desired.